### PR TITLE
Update extensions pages

### PR DESCRIPTION
**Pull request by Bouh**
- Simplify some stuff
- Add references of parameters types
- Explain what is for the Identifier parameter.

### CODE REVIEW

**Comment by Bouh** [comment]
@AlexandreSi I wonder if I have the right to edit `/gdevelop5/extensions/index.md` because I edited the text in the top of the page, but I know one part of the page is generated.
So I don't know if my changes will be erased at the next generation of the page.

File: docs/gdevelop5/events/functions/index.md [hunk 1]
```diff
@@ -146,4 +146,40 @@ Functions can be shared across projects (like actions/conditions that are built
...
+
+
+### Identifier parameter
+
+This parameter enables auto-completion for previously entered text, helping users quickly locate values from other events.
+
+#### Scopes
+
+- **Object**: Suggests previously used values on the same object.
+- **Scene**: Suggests previously used values from the scene events.
```
Comment (D8H) [inline docs/gdevelop5/events/functions/index.md:181]: It should probably be infinitive as there is no subject.
```suggestion
- **Object** - To suggest previously used values on the same object.
- **Scene** - To suggest previously used values from the scene events.
```

File: docs/gdevelop5/events/functions/index.md [hunk 2]
```diff
@@ -146,4 +146,40 @@ Functions can be shared across projects (like actions/conditions that are built
...
+
+#### Identifier name
+
+Is an unique name used by the auto-completer to reterive similar values used by the user.
```
Comment (D8H) [inline docs/gdevelop5/events/functions/index.md:185]:
```suggestion
Every parameter with the same identifier name share their values for the autocompletion.
```

File: docs/gdevelop5/events/functions/index.md [hunk 3]
```diff
@@ -146,4 +146,40 @@ Functions can be shared across projects (like actions/conditions that are built
...
+| Yes or No                     | Boolean      | A boolean value                                           |
+| True or False                 | Boolean      | A boolean value                                           |
+| Object point                  | Text         | The name of an point from an object                       |
+| Object animation              | Text         | The name of an animation from an object                   |
+| Identifier                    | Text         | Allow the auto-complete to suggest previously used values |
+
+
+### Identifier parameter
+
+This parameter enables auto-completion for previously entered text, helping users quickly locate values from other events.
```
Comment (D8H) [inline docs/gdevelop5/events/functions/index.md:176]: I don't understand this part "helping users quickly locate values from other events".
What does "locate" mean?

File: docs/gdevelop5/events/functions/index.md [hunk 4]
```diff
@@ -146,4 +146,40 @@ Functions can be shared across projects (like actions/conditions that are built
...
-    See [an example of replacing external events by a function](/gdevelop5/events/functions/replace-external-events), which is reusable and shareable. You can also **[automatically extract events to a function](/gdevelop5/events/functions/extract-events)**.
\ No newline at end of file
+    See [an example of replacing external events by a function](/gdevelop5/events/functions/replace-external-events), which is reusable and shareable. You can also **[automatically extract events to a function](/gdevelop5/events/functions/extract-events)**.
+
+
+## References of parameters
+
+Parameters in function can be various:
+
+| Parameters                    | Type         | Description                                               |
```
Comment (D8H) [inline docs/gdevelop5/events/functions/index.md:156]: I don't know how the 1st 2 columns must be named.
```suggestion
| Type                    | Type         | Description                                               |
```

I'm not sure the table is useful. The description doesn't give much info compare the UI.

File: docs/gdevelop5/events/functions/index.md (hunk 3 above)
Comment (Bouh) [reply docs/gdevelop5/events/functions/index.md:176]: It was for locate/found/see the values already used in other events.

File: docs/gdevelop5/events/functions/index.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ Add a parameter with the button "Add a parameter":
 For each parameter, you can enter:
 
   * The name, that will be used to access the parameter in the *events of the function*.
-  * The type. The parameter can be objects, a number or a text. If you choose objects, you'll also be able to choose the type of object to receive.
+  * The type. The parameter can be objects, a number, a text, etc. [See the complete list of parameters](/gdevelop5/events/functions/#references-of-parameters).
```
Comment (D8H) [inline docs/gdevelop5/events/functions/index.md:66]:
```suggestion
  * The type. Parameters can be objects, numbers, texts, etc. [See the complete list of parameters](/gdevelop5/events/functions/#references-of-parameters).
```

File: docs/gdevelop5/events/functions/index.md [hunk 6]
```diff
@@ -146,4 +146,40 @@ Functions can be shared across projects (like actions/conditions that are built
...
+
+Parameters in function can be various:
+
+| Type                          | Data type    | Description                                               |
+|-------------------------------|--------------|-----------------------------------------------------------|
+| [Objects](/gdevelop5/objects) | Objects list | A list of objects                                         |
+| [Behavior](/gdevelop5/behaviors) | Behavior     | The behavior from an object                               |
+| String                        | Text         | Just a text                                               |
+| String from a list of options | Text         | A list of multiple text                                   |
+| Keyboard Key                  | Text         | A list of keys                                            |
```
Comment (D8H) [inline docs/gdevelop5/events/functions/index.md:162]:
```suggestion
| Keyboard Key                  | Text         | Let extension users choose a key from a list                                           |
```

File: docs/gdevelop5/extensions/index.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
 # Extensions
 
-GDevelop is built in a flexible way. In addition to [core features](/gdevelop5/all-features), new capabilities are provided by extensions. Extensions can contain objects, behaviors, actions, conditions, expressions or events.
+GDevelop is built in a flexible way. In addition to [core features](/gdevelop5/all-features), new capabilities are provided by extensions.
+Extensions can contain objects, behaviors, actions, conditions and expressions. They can be installed or made by anyone directly inside GDevelop.
```
Comment (D8H) [inline docs/gdevelop5/extensions/index.md:4]: Is this line break intended?

Comment (Bouh) [inline docs/gdevelop5/extensions/index.md:4]:
```suggestion
GDevelop is built in a flexible way. In addition to [core features](/gdevelop5/all-features), new capabilities are provided by extensions. Extensions can contain objects, behaviors, actions, conditions and expressions. They can be installed or made by anyone directly inside GDevelop.
```

**Pull request merged**